pootle/apps/pootle_store/models.py

```diff
@@ -211,6 +210,9 @@ def stringcount(string):
         return 1
 
 
+TMServer = SearchBroker()
```

Comment:
The thinking is that TM is just a specialised search and we want a general search abstraction.  So I'd prefer to leave it as this for now.